src/urls.ts

```diff
@@ -4,6 +4,7 @@ export const testingEditedPackages = `${baseURL}#testing`;
 export const testingNewPackages = `${baseURL}#testing`;
 export const definitionOwners = `${baseURL}#definition-owners`;
 export const workflow = `${baseURL}#make-a-pull-request`;
+export const otherFilesTxt = `${baseURL}#other_filestxt`;
```

Comment:
This will need to be rewritten with the #284, using the section title.

Reply:
Done :heavy_check_mark: